mkdocs.yml

```
@@ -0,0 +1,130 @@
+      emoji_generator: !!python/name:material.extensions.emoji.to_svg
+  - attr_list
+  - def_list
+  - md_in_html
+  - toc:
+      permalink: true
+
+# Navigation
+nav:
+  - Home: index.md
+  - Getting Started:
+    - Overview: getting-started/overview.md
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)